coralproject/talk
# [CORL-1323] Media Fixes

## What does this PR do?

- Fixes a bug where media is selected and deselected it not actually removing it from the component
- Fixes a bug where an invalid url is previewed
- Fixes a bug where the media field would show multiple options instead of only one
- Fixes a bug where the edit field removes the external image



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



1. Try to attach media, see that you can switch modes easily!
2. Try to edit a comment with an image, see that it works!
